app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for person in people:
+        select_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            select_person.wife = Person.people[person["wife"]]
```

Comment:
The code assumes that the 'wife' is already created and exists in the `Person.people` dictionary. If the 'wife' is not created before this line is executed, it will raise a `KeyError`. Consider adding a check to ensure the referenced person exists.